framework/web/js/source/jquery.ba-bbq.min.js

```diff
@@ -1,18 +1,17 @@
 /*
  * jQuery BBQ: Back Button & Query Library - v1.3pre - 8/26/2010
- * https://benalman.com/projects/jquery-bbq-plugin/
+ * http://benalman.com/projects/jquery-bbq-plugin/
```

Comment:
Could you update `v1.3pre` to `v1.4pre`? (was already the case but documented wrongly)
And please revert the 4 lines that change `https` to `http`